dev-tools/cmd/dashboards/export_dashboards.go

```diff
@@ -157,42 +52,55 @@ func main() {
 	flag.Parse()
 	log.SetFlags(0)
 
-	transCfg := &http.Transport{
-		TLSClientConfig: &tls.Config{InsecureSkipVerify: true}, // ignore expired SSL certificates
+	u, err := url.Parse(*kibanaURL)
+	if err != nil {
+		log.Fatalf("Error parsing Kibana URL: %v", err)
```

Comment:
I have a mistake, `url.Parse` or `url.ParseRequestURL` is well really bad see for yourself:

https://play.golang.org/p/9NAU0-sasfT

I think we have to come up with something better, maybe we already have in other outputs.

Reply:
In elasticsearch output `url.Parse` is used. Looking at the codebase, everytime we connect to something `url.Parse` is used. I think it is good enough.